signup/signup.css

```diff
@@ -0,0 +1,184 @@
+body
+{
+	  margin: 0px;
+    padding: 0px;
+    background-image: url("./images/budget_signup.jpg");
+		background-position-x: -145px;
+}
+
+/*SIGNUP FRAME*/
+#signup
+{
+    width: 320px;
+    height: 450px;
+    background-color:RGB(0,0,0,0.5);
+    margin: auto;
+    margin-top: 65px;
+    padding: 20px;
+    padding-top: 4px;
+    color: white;
+    border-radius: 50px;
+}
+
+.budget
+{
+    text-align: center;
+    margin: 0px;
+    padding: 5px;
+    font-family: cursive;
+}
+
+.budget_txt
+{
+    margin: 0px;
+}
+
+.budget_txt:after
+{
+    position: absolute;
+    content: "BUDGET";
+    color: red;
+    font-family: cursive;
+    font-size: 1.0em;
+    margin-top: -1px;
+    margin-left: -132px;
+}
+
+.sign_up
+{
+    text-align: center;
+    text-indent: -34px;
+    font-family: cursive;
+    margin: 0px;
+    padding: 5px;
+}
+
+.sign_up_txt
+{
+    margin: 0px;
+}
+
+.sign_up_txt:before
+{
+    position: absolute;
+    content: " ";
+    color: red;
+    border-bottom: 2px solid red;
+    width: 100px;
+    margin-top: 30px;
+}
+
+.sign_up_txt:after
+{
+    position: absolute;
+    content: "UP";
+    color: red;
+    margin-left: 35px;
+    font-family: cursive;
+    font-size: 1.1em;
+    margin-top: -2px;
+}
+
+form
+{
+	  margin-left:10px;
+}
+
+.int_submit
+{
+    margin: 5px;
+    padding: 5px;
+    text-align: center;
+}
+
+.submit
+{
+    width: 192px;
+    height: 29px;
+    text-align: center;
+    border-radius: 9px;
+    box-shadow: 0px 1px 5px 0px;
+    font-size: 20px;
+    font-family: cursive;
+    font-style: italic;
+    font-weight: bold;
+    border: 2px solid black;
+    padding-bottom: 30px;
+}
+
+input , .address_txt
+{
+    margin: 3px;
+    border-radius: 5px;
+    border: 2px inset #353e44;
+    color: white;
+    background-color: RGB(182,58,60,0.5);
+    box-shadow: 0px 1px 5px 0px;
+	  height: 25px
+}
+
+.title_selc
+{
+    margin: 3px;
+    margin-left: 81px;
+    color: black;
+    border: 2px inset #353e44;
+    border-radius: 5px;
+    background-color: RGB(182,58,60,0.5);
+    box-shadow: 0px 1px 5px 0px;
+}
+
+.name_txt
+{
+    margin-left: 60px;
+    width: 190px;
+}
+
+.dob_selc
+{
+    margin-left: 17px;
+    width: 190px;
+}
+
+.emailid_txt
+{
+    margin-left: 42px;
+  	width: 190px;
+}
+
+.contact_number_txt
+{
+    margin-left: 21px;
+	  width: 190px;
```

Comment:
we can create one class which have width of 190px and add that class to every element that wants to use that.